src/System.Private.CoreLib/shared/System/Tuple.cs

```diff
@@ -261,7 +225,7 @@ public override int GetHashCode()
 
         int IStructuralEquatable.GetHashCode(IEqualityComparer comparer)
         {
-            return Tuple.CombineHashCodes(comparer.GetHashCode(m_Item1!), comparer.GetHashCode(m_Item2!));
+            return HashCode.Combine(comparer.GetHashCode(m_Item1!), comparer.GetHashCode(m_Item2!));
```

Comment:
This would break F#: https://github.com/dotnet/coreclr/blob/46f2b52d2d5b148ec026f7bd011f38f63824dd2e/src/System.Private.CoreLib/shared/System/Tuple.cs#L12

Reply:
Awesome.